docs/prod-install.md

```diff
@@ -12,13 +12,13 @@ you need:
 * At least 2GB RAM and 10 GB disk space (4GB and 2 CPUs recommended for 100+ users).
 * A DNS name, an SSL certificate, and credentials for sending email.
 
-## Step 0: Subscribe
+## Step 1: Subscribe
```

Comment:
I think "Step 0" is just strange.  I think it's valid to break out a prep section where we list steps like a, b, and c.  The current "Step 0" is actually kind of bogus; we should have people log on to chat to get help.  If anything "Step 0" should actually be the last step.